src/components/transactions/Bridge/useGetBridgeMessage.tsx

```diff
@@ -43,6 +45,8 @@ export const useGetBridgeMessage = ({
       //     transfer is supported.
       // ==================================================
       // */
+
+      // TODO: Do we need this? Right now ONLY GHO Supported
```

Comment:
hmm yeah probably not since we have the bridge config which will contain all of the supported networks

Reply:
yeah thinking we start to review the bridge branch with main and start to make a list of things